examples/webgl_occlusionqueries.html

```diff
@@ -0,0 +1,229 @@
+
+				// EVENTS
+				window.addEventListener( 'resize', onWindowResize, false );
+
+				// CONTROLS
+				cameraControls = new THREE.OrbitControls( camera, renderer.domElement );
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0xAAAAAA );
+
+				scene.add( ambientLight );
+				scene.add( light );
+
+			}
+
+
+			function onWindowResize() {
+
+				var canvasWidth = window.innerWidth;
+				var canvasHeight = window.innerHeight;
+
+				renderer.setSize( canvasWidth, canvasHeight );
+
+				camera.aspect = canvasWidth / canvasHeight;
+				camera.updateProjectionMatrix();
+
+				render();
+
+			}
+
+
+			function render() {
+
+
+				renderer.render( scene, camera );
+
+			}
+
+			var sphere;
+			function createScene() {
+
+
+				var ground = new THREE.Mesh(new THREE.PlaneBufferGeometry(100, 100, 10, 10), new THREE.MeshStandardMaterial({ color: 0x22CC66, side: THREE.DoubleSide }));
+				ground.receiveShadow = true;
+				ground.position.set(0, -5, 0);
+				ground.rotation.x = -Math.PI / 2;
+				scene.add(ground);
+
+				sphere = new THREE.OcclusionQueryMesh(new THREE.SphereBufferGeometry(1, 15, 15), new THREE.MeshStandardMaterial({
+					color: 0xFF4411,
+					// disabling colorWrite should not break the occlusion testing
+					colorWrite: true
+				}));
+				sphere.frustumCulled = false;
+				sphere.receiveShadow = true;
+				sphere.castShadow = true;
+				sphere.renderOrder = 999
+				var highestFrame = -1;
+				var shadowHighestFrame = -1;
+				sphere.occlusionQueryCallback = function ( result, frame, queryCamera ) {
+
+					var text;
+					if ( queryCamera === camera && frame > highestFrame ) {
+
+						highestFrame = frame;
+						var text = result ? 'Sphere visible onscreen' : 'Sphere occluded onscreen';
+						if ( queryResult.innerText !== text ) { queryResult.innerText = text; }
+
+					}
+					if ( queryCamera === light.shadow.camera && frame > shadowHighestFrame ) {
+
+						shadowHighestFrame = frame;
+						text = result ? 'Sphere visible in shadowmap' : 'Sphere occluded in shadowmap';
+						if ( shadowQueryResult.innerText !== text ) { shadowQueryResult.innerText = text; }
+
+					}
+
+				};
+
+				var mat = new THREE.MeshStandardMaterial({ color: 0x6666AA });
+				var leftBox = new THREE.Mesh(new THREE.TorusKnotBufferGeometry( 2, 0.8, 128, 16 ), mat);
+				leftBox.receiveShadow = true;
+				leftBox.castShadow = true;
+				leftBox.position.set(-5, 0, -3);
+				scene.add(leftBox);
+				var rightBox = new THREE.Mesh(new THREE.BoxBufferGeometry(5, 9, 1), mat);
+				rightBox.receiveShadow = true;
+				rightBox.castShadow = true;
+				rightBox.position.set(5, 0, 3);
+				scene.add(rightBox);
+
+				scene.add(sphere);
+
+				// var helper = new THREE.CameraHelper( light.shadow.camera );
+				// scene.add( helper );
+
+			}
+
+			init();
+
+			createScene();
+
+			var t = 0;
+			var tick = function (t) {
+				sphere.position.x = Math.sin(t / 1000) * 5
+				render();
+				requestAnimationFrame(tick);
+			};
+			tick(1);
```

Comment:
Please make an `animate()` function like in all other examples. Also use `THREE.Clock` if you need a time value. Something like:
```js
function animate() {

    requestAnimationFrame( animate );

    var t = clock.getElapsedTime();

    sphere.position.x = Math.sin( t / 1000 ) * 5;

    render();

}